chingu-coders/Voyage2-Turtles-02
# Minor bug for re-sizable icon

## Purpose
* Fix minor bug that has sticky note re-sizable icon appearing above other elements

## Approach/Fixes
* Changes z-index to 0

## Testing
* Download this PR to your local machine
* Test that the icon is ONLY seen on it's current sticky note
* Try putting the sticky note behind other elements and check to see that the re-sizable icon doesn't persist above those elements
